ext/RMagick/extconf.rb

```diff
@@ -343,6 +306,76 @@ def print_summary
       message summary
     end
   end
+
+  class Config
+    include Singleton
+
+    def initialize
+      @windows = (RUBY_PLATFORM =~ /mswin|mingw/)
```

Comment:
Can we make this a strict boolean? Otherwise it'll be `nil` or an integer.

```suggestion
      @windows = !!(RUBY_PLATFORM =~ /mswin|mingw/)
```